en/patterns/details.md

```diff
@@ -12,17 +12,24 @@ Use the Details Pattern to display detailed information as a full-screen page us
 
 The Details Pattern comes with the styling flexibility provided by the Badges, Buttons, Tabs, and Text elements that constitute it.
 
-### Content Types
+### Details: Icon + Title 
```

Comment:
Remove the Details: from here and from the paragraph. Leave it just to say Icon + Title